ci/common.groovy

```diff
@@ -71,6 +71,8 @@ def prep(type = 'nightly') {
   }
   /* node deps, pods, and status-go download */
   utils.nix.shell("scripts/prepare-for-platform.sh ${prepareTarget}", pure: false)
+  utils.nix.shell("cp -R translations status-modules/translations")
```

Comment:
i would merge these 2 lines together, so that the build is faster (by not waiting to instantiate a Nix shell each time). Even better, no Nix shell since these are only file copies:
```suggestion
  sh("cp -R translations status-modules/translations && cp -R status-modules node_modules/status-modules")
```

Reply:
ok, I will change it now